### PR TITLE
Cleanup server tests

### PR DESCRIPTION
**Pull request by davidegrohmann**

### CODE REVIEW

File: community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java [hunk 1]
```diff
@@ -1,108 +0,0 @@
...
-
-import org.neo4j.graphdb.Node;
-import org.neo4j.kernel.GraphDatabaseAPI;
-import org.neo4j.server.WrappingNeoServerBootstrapper;
-import org.neo4j.server.configuration.Configurator;
-import org.neo4j.server.configuration.ServerConfigurator;
-import org.neo4j.test.ImpermanentGraphDatabase;
-
-@Ignore("This feature has to be done some other way")
-public class ExecutionTimeLimitTest
```
Comment (jakewins) [inline community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java:37]: We should add a card to replace this test (and one to re-implement this feature).

**Comment by jakewins** [comment]
This looks awesomesauce imo

File: community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java (hunk 1 above)
Comment (davidegrohmann) [inline community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java:37]: You probably know better than me what to write in those cards. Mind to create them?

Comment (jakewins) [inline community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java:37]: +1, I'll add it to the remoting wall

**Pull request merged**